draft-ietf-mls-extensions.md

```diff
@@ -1472,20 +1456,39 @@ from a group more efficiently than using a `remove` proposal type, as the
 * Recommended: Y
 * Reference: RFC XXXX
 
-<!-- ## MLS Signature Labels
+## MLS Signature Labels
+
+### TargetedMessageTBS
+
+* Label: "TargetedMessageTBS"
+* Recommended: Y
+* Reference: RFC XXXX
 
-### Labeled Extension Content
+### CredentialBindingTBS
 
-* Label: "LabeledExtensionContent" (suggested)
+* Label: "CredentialBindingTBS"
 * Recommended: Y
-* Reference: RFC XXXX -->
+* Reference: RFC XXXX
 
 ## MLS Component Types {#iana-components}
 
 This document requests the creation of a new IANA "MLS Component Types" registry under the "Messaging Layer Security" group registry heading. Assignments to this registry in the range 0x0000 0000 to 0x7FFF FFFF are via Specification Required
 policy {{!RFC8126}} using the MLS Designated Experts. Assignments in the range
 0x8000 0000 to 0xFFFF FFFF are for private use.
 
+As described in {{Section 13.5 of !RFC9420}}, GREASE values are ignored by
+receivers, while a random selection of GREASE values are included in certain
+fields. This specification adds the following locations where GREASE values for
+components can be included:
```

Comment:
I think this paragraph should use normative language.

Reply:
I agree with with normative language.  I would suggest that GREASE SHOULD be added to these fields _if present_.  To emphasize that you don't need to add these fields just to send GREASE.